src/app.js

```diff
@@ -1 +1,59 @@
 'use strict';
+
+const { numberGenerator } = require('./numberGenerator');
+const { bullsAndCowsCalculator } = require('./bullsAndCowsCalculator');
+const { messages } = require('./messages');
+const { terminal } = require('./terminal');
+
+const [
+  introMessage,
+  congratsMessage,
+  gameOverMessage,
+  digitsError,
+  numberError,
+] = messages();
+
+terminal.question(introMessage, (userInput) => {
+  if (userInput === 'yes') {
+    let guesses = 0;
+    const maxGuesses = 10;
+    const numberToGuess = numberGenerator();
+
+    const guessNumber = () => {
+      terminal.question('\nEnter your guess: ', (guess) => {
+        if (guess.length !== 4) {
+          terminal.write(numberError);
+          guessNumber();
+
+          return;
+        }
+
+        if (!/^[1-4]+$/.test(guess)) {
```

Comment:
Look at example. There aren't only digits from 1 to 4. digits can be from 0 to 9. Otherwise you don't actualy need 'cows' for this game, as it tells you if some digit is in the number